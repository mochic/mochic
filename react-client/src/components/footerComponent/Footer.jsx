import React, { Component } from 'react';

class Footer extends Component {
    render() {
        return (
            <footer>
                <div>footer component</div>
                <a href="https://github.com/mochic/mochic">repo</a>
            </footer>
        )
    }
}

export default Footer;
