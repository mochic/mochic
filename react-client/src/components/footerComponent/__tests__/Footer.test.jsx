import React from 'react';
import Footer from '../Footer';
import renderer from 'react-test-renderer';

it('renders deep', () => {
  const tree = renderer.create(<Footer />);
  expect(tree).toMatchSnapshot();
});
