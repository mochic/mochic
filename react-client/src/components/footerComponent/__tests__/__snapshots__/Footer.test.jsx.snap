// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders deep 1`] = `
<footer>
  <div>
    footer component
  </div>
  <a
    href="https://github.com/mochic/mochic"
  >
    repo
  </a>
</footer>
`;
